### ISSUE TITLE
New HTML 5 Reg Chart won't display in Safari on iOS . 

### ISSUE THREAD

**Issue by JonathanNawrocki** [rjenks/Con-Org#81]
After waiting several minutes for the pre Reg chart page to load on an iPad and iPhone 6s, the screen never draws out the chart.
![image](https://cloud.githubusercontent.com/assets/14830893/13417004/3857347a-df2d-11e5-8a54-c2c14b93acbf.png)

**Comment by rjenks** [comment]
I'm not sure if this is the issue, but I just had the reg chart stop working for me in Chrome.  I found an error in the Javascript console and looked it up.  It appears there is a bug in the latest version of Google Charts and the workaround it loading an old version.  I just changed the code and redeployed.  See if that works.

details: http://stackoverflow.com/questions/35648530/google-visualization-charts-api-examples-are-broken-how-to-fix-them
